wordpress-mobile/WordPress-iOS
# Issue/11397 Fix Zendesk presentation crash

Fixes #11397. I think this PR should fix the crash we're seeing in #11397. It switches Zendesk to use a modal presentation style instead of pushing onto the existing navigation stack. We're using `formSheet`, which will show full screen on iPhone, and in a smaller modal on iPad.

![before-after-zendesk](https://user-images.githubusercontent.com/4780/62454794-e2be8b00-b76c-11e9-8f42-93a581c0e70f.png)

**To test:**

* Build and run
* Ensure that Zendesk views still work as expected
* Create a new ticket, and tap the attachment icon. Scroll the image picker to the left and choose Media Library or Camera and ensure the app doesn't crash.

cc @jkmassel @SylvesterWilmott 

Update release notes:
